src/coverlet.console/Logging/ConsoleLogger.cs

```diff
@@ -1,39 +1,39 @@
 using System;
-using static System.Console;
+using Coverlet.Core.Logging;
+using SystemConsole = System.Console;
```

Comment:
Is this necessary?I'm OOF and I cannot try with VS is there a name collision somewhere?

Reply:
Before it was using static namespace.   Which causes a ambiguity with the locally defined WriteLine method.